src/v3/src/components/Checkbox/Checkbox.tsx

```diff
@@ -10,121 +10,70 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { useOdysseyDesignTokens } from '@okta/odyssey-react-mui';
-import {
-  Box,
-  Checkbox as CheckboxMui,
-  FormControl,
-  FormControlLabel,
-  FormHelperText,
-} from '@okta/odyssey-react-mui-legacy';
-import { Fragment, h } from 'preact';
+import { Checkbox as OdyCheckbox, CheckboxGroup } from '@okta/odyssey-react-mui';
+import { h } from 'preact';
 
 import { useWidgetContext } from '../../contexts';
-import { useAutoFocus, useValue } from '../../hooks';
+import { useAutoFocus, useHtmlContentParser, useValue } from '../../hooks';
 import {
   ChangeEvent,
   UISchemaElementComponent,
   UISchemaElementComponentWithValidationProps,
 } from '../../types';
-import { getTranslationInfo } from '../../util';
-import FieldLevelMessageContainer from '../FieldLevelMessageContainer';
+import { buildFieldLevelErrorMessages, getTranslationInfo, wrapInTranslateNo } from '../../util';
 import { withFormValidationState } from '../hocs';
 
 const Checkbox: UISchemaElementComponent<UISchemaElementComponentWithValidationProps> = ({
   uischema,
   errors,
   handleChange,
   handleBlur,
-  describedByIds,
 }) => {
   const value = useValue(uischema);
   const { loading } = useWidgetContext();
 
   const {
     options: { inputMeta: { name, mutable } },
     focus,
-    noTranslate,
     translations = [],
-    showAsterisk,
   } = uischema;
   const isReadOnly = mutable === false;
+  const checkboxId = `${name}-checkbox`;
   const labelInfo = getTranslationInfo(translations, 'label');
+  const label = useHtmlContentParser(labelInfo?.noTranslate
+    ? wrapInTranslateNo(labelInfo.value)
+    : labelInfo?.value) as string | undefined;
   const descriptionInfo = getTranslationInfo(translations, 'description');
+  const description = useHtmlContentParser(descriptionInfo?.noTranslate
+    ? wrapInTranslateNo(descriptionInfo.value)
+    : descriptionInfo?.value) as string | undefined;
   const focusRef = useAutoFocus<HTMLInputElement>(focus);
-  const hasErrors = typeof errors !== 'undefined';
-  const tokens = useOdysseyDesignTokens();
+  const { errorMessage, errorMessageList } = buildFieldLevelErrorMessages(errors);
 
   return (
-    <FormControl
-      component="fieldset"
-      error={hasErrors}
-      aria-describedby={describedByIds}
-      className={noTranslate ? 'no-translate' : undefined}
+    <CheckboxGroup
+      errorMessage={errorMessage}
+      errorMessageList={errorMessageList}
+      id={name}
+      label=""
```

Comment:
Because of the granular consent views, we have situations where individual checkboxes can get errors rendered under them that are specific to that checkbox. Thus, we have to wrap each checkbox in an Odyssey `CheckboxGroup` so we can pass in error messages because the individual `Checkbox` component in Odyssey does not support rendering error messages below it